text/0000-nuke-return-this.md

```diff
@@ -0,0 +1,158 @@
+  - `upper_in_place`: will return `None`
+  - `reverse_in_place`: will return `None`
+  - `push`: will return `None`
+  - `unshift`: will return `None`
+  - `append`: will return `None`
+  - `concat`: will return `None`
+  - `clear`: will return `None`
+  - `insert_in_place`: will return `None` (`String.insert` and `Array.insert` are inconsistent. `String.insert_in_place` does the same thing as `Array.insert`)
+  - `insert_byte`: will return `None`
+  - `cut_in_place`: will return `None`
+  - `replace`: will return the number of occurrences replaced
+  - `strip`: will return `None`
+  - `lstrip`: will return `None`
+  - `rstrip`: will return `None`
+- `buffered.Reader`
+  - `clear`: will return `None`
+  - `append`: will return `None`
+  - `skip`: will return `None`
+- `buffered.Writer`
+  - `reserve`: will return `None`
+  - `reserve_chunks`: will return `None`
+  - number write functions (e.g. `u16_le`): will all return `None`
+  - `write`: will return `None`
+  - `writev`: will return `None`
+- `capsicum.CapRights0`
+  - `set`: will return `None`
+  - `unset`: will return `None`
+- `collections.Flag`
+  - `all`: will return `None`
+  - `clear`: will return `None`
+  - `set`: will return `None`
+  - `unset`: will return `None`
+  - `flip`: will return `None`
+  - `union`: will return `None`
+  - `intersect`: will return `None`
+  - `difference`: will return `None`
+  - `remove`: will return `None`
+- `collections.ListNode`
+  - `prepend`: will return whether the node was already in a `List`
+  - `append`: will return whether the node was already in a `List`
+  - `remove`: will return `None`
+- `collections.List`
+  - `reserve`: will return `None`
+  - `remove`: will return the removed element (the method is inconsistent with `Array.remove`)
+  - `clear`: will return `None`
+  - `prepend_node`: will return `None`
+  - `append_node`: will return `None`
+  - `prepend_list`: will return `None`
+  - `append_list`: will return `None`
+  - `push`: will return `None`
+  - `unshift`: will return `None`
+  - `append`: will return `None`
+  - `concat`: will return `None`
+  - `truncate`: will return `None`
+- `collections.Map`
+  - `concat`: will return `None`
+  - `compact`: will return `None`
+  - `clear`: will return `None`
+- `collections.RingBuffer`
+  - `push`: will return whether the collection was full
+  - `clear`: will return `None`
+- `collections.Set`
+  - `clear`: will return `None`
+  - `set`: will return `None`
+  - `unset`: will return `None`
+  - `union`: will return `None`
+  - `intersect`: will return `None`
+  - `difference`: will return `None`
+  - `remove`: will return `None`
+- `files.FileMode`
+  - `exec`: will return `None`
+  - `shared`: will return `None`
+  - `group`: will return `None`
+  - `private`: will return `None`
+- `files.File`
+  - `seek_end`: will return `None`
+  - `seek`: will return `None`
+  - `flush`: will return `None`
+  - `sync`: will return `None`
+- `options.Options`
+  - `add`: will return `None`
```

Comment:
The current signature of this function is `fun ref add(long: String, short: (None | String) = None, arg: ArgumentType = None, mode: (Required | Optional) = Required): Options`. I don't think `add` is meant as the binary operator sugar here. I'm not sure if another name would make more sense.